applications/ShapeOptimizationApplication/tests/algorithm_steepest_descent_test/parameters.json

```diff
@@ -56,19 +56,20 @@
         "optimization_algorithm" : {
             "name"               : "steepest_descent",
             "max_iterations"     : 20,
-            "relative_tolerance" : 1e-0,
+            "relative_tolerance" : 1e-2,
+            "gradient_tolerance" : 0.8,
             "line_search" : {
-                "line_search_type"           : "manual_stepping",
+                "line_search_type"           : "adaptive_stepping",
```

Comment:
can you create a new test for the 'adaptive_stepping' and leave this one as is?

Reply:
Yep.